tests/parm/ufs.configure.s2s.IN

```diff
@@ -3,8 +3,8 @@
 #############################################
 
 # ESMF #
-logKindFlag:            ESMF_LOGKIND_MULTI
-globalResourceControl:  true
+logKindFlag:            @[esmf_logkind]
+globalResourceControl:  false
```

Comment:
If the esmf managed threading is off (by setting globalresourcecontrol false), are the various omp_num_threads attributes ignored? Because if they are, we would not need the esmf/non-esmf versions.

@junwang-noaa Do you know if that might work?

Reply:
@junwang-noaa and I had that exact same realization. She is going to investigate

Reply:
I ran a test case, unfortunately, it does not work. The error message:

  0:  Starting PEs :           30
  0:  Starting Threads :            2
  0:
  0: FATAL from PE     0: mpp_domains_define.inc: not all the pe_end are in the pelist

I guess the component still reads in the number of threads

Reply:
@junwang-noaa Thanks for the test and confirmation.  Perhaps we could make a feature request for this in the future and eliminate the duplication of the `ufs.configure` templates threading and consolidate.

Reply:
@junwang-noaa How did you test this? I had some run directories on hercules, and for control_p8 and control_2threads_p8, I change the resourceflag to false and both ran. I did the same w/ the cpld_2threads_p8 and it also ran.